src/pages/build-apps/guides/authentication.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Authentication
+description: Register and sign in users with identities on the Stacks blockchain
+experience: beginners
+tags:
```

Comment:
let's remove the "experience" and "tags" - they suggest a tutorial. this seems also to be present in the other guides